### ISSUE TITLE
DOC: documentation of Netscript functions won't build

### ISSUE THREAD

**Issue by quacksouls** [bitburner-official/bitburner-src#160]
On the dev branch 239834dba61f71078cd5b345f40ddb3193663e74.  Building the documentation of Netscript functions resulted in this:
```sh
$ npm run doc

> bitburner@2.1.0 doc
> npx api-extractor run && npx api-documenter markdown && rm input/bitburner.api.json && rm -r input


api-extractor 7.19.4  - https://api-extractor.com/

Using configuration from ./api-extractor.json
Analysis will use the bundled TypeScript version 4.5.5
Warning: src/ScriptEditor/NetscriptDefinitions.d.ts:6876:1 - (ae-missing-release-tag) "EmployeePosName" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
Warning: src/ScriptEditor/NetscriptDefinitions.d.ts:6885:1 - (ae-missing-release-tag) "IndustryTypeName" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)

API Extractor completed with warnings
```
The Markdown files of the documentation are not generated at all.

**Comment by Snarling** [comment]
This is my bad, wasn't sure the requirements for including `@public` tag.

**Comment by Snarling** [comment]
Fixed by 4bf215f
